src/NetworkSettings.cpp

```diff
@@ -2,12 +2,13 @@
 /*
  * Copyright (C) 2022 Thomas Basler and others
  */
-#include "NetworkSettings.h"
 #include "Configuration.h"
 #include "MessageOutput.h"
+#include "NetworkSettings.h"
```

Comment:
The header file belonging to the cpp file should stay in the first line (this is also done by the auto formatter of vscode)

Reply:
Normally I would have a empty line between .h file of a .cpp file and all other headers :P